src/rpc/blockchain.cpp

```diff
@@ -1597,7 +1597,12 @@ static UniValue invalidateblock(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_DATABASE_ERROR, FormatStateMessage(state));
     }
 
-    return NullUniValue;
+    UniValue res(UniValue::VOBJ);
+    {
+        LOCK(cs_main);
+        res.pushKV("invalidated", !chainActive.Contains(pblockindex));
```

Comment:
Need to be documented in the RPC help. Please also mention in what cases it can happen that the block was not invalidated but it does get all the way to the end here.